app/views/content/blog/a-diary-of-a-trainee-teacher.md

```diff
@@ -31,9 +31,9 @@ When lockdown hit in 2020, I began to reassess my career choices as I knew I had
 
 ## The first few days
 
-My course started on 1st September, and I spent the first three days with my School Direct partner, Sheffield Teacher Training Alliance. This was a fantastic opportunity to get to know the rest of the cohort, including students training to teach other subjects on the curriculum. Sheffield Teacher Training Alliance had organised various sessions over these three days which were linked to the [Initial Teacher Training Core Content Framework](https://www.gov.uk/government/publications/initial-teacher-training-itt-core-content-framework). 
+My course started on 1st September, and I spent the first three days with Sheffield Teacher Training Alliance. This was a fantastic opportunity to get to know the rest of the cohort, including students training to teach other subjects on the curriculum. Sheffield Teacher Training Alliance had organised various sessions over these three days which were linked to the [Initial Teacher Training Core Content Framework](https://www.gov.uk/government/publications/initial-teacher-training-itt-core-content-framework). 
```

Comment:
know this is what it's like in production, just wondered if we wanna tweak while we're here so it's more in line with the gov.uk style guide!
```suggestion
My course started on 1 September, and I spent the first 3 days with Sheffield Teacher Training Alliance. This was a fantastic opportunity to get to know the rest of the cohort, including students training to teach other subjects on the curriculum. Sheffield Teacher Training Alliance had organised various sessions over these 3 days which were linked to the [Initial Teacher Training Core Content Framework](https://www.gov.uk/government/publications/initial-teacher-training-itt-core-content-framework). 
```